lib/fetch-libc++.sh

```diff
@@ -19,14 +19,21 @@ MIRRORS=(\
   "https://alpha.mirror.svc.schuermann.io/files/tock"\
 )
 
+if test -x /usr/bin/shasum; then
```

Comment:
This _at least_ needs a comment. Why this particular path? Do we always prefer `shasum` if it is present? Or only if `sha256sum` _isn't_ present?

Is this the `shasum` that comes with Perl? If so, Perl is a pre-requisite for most Linux distributions in practice, does it make sense to just use `shasum`?

Reply:
This is what happened in the kernel: https://github.com/tock/tock/pull/1669. Should we have done something differently? I think we just want this to work on as many systems as possible.